index.html

```diff
@@ -8,6 +8,75 @@
+                <div class="swatch-one"></div>
+                <input id="foreground" name="foreground" class="form-input" type="text" value="#000000" />
+              </div>
+            </div>
+            <div class="input-fields">
+              <div class="form-item">
+                <label for="background">Background color</label>
+              </div>
+              <div class="input-items">
+                <div class="swatch-two"></div>
+                <input id="background" name="background" class="form-input" type="text" placeholder="Second color" />
+              </div>
+            </div>
+          </div>
+          <div class="result-row">
+            <div class="ratio-result">
+              <div>
+                <p>Contrast Ratio</p>
+                <p class="large">4.51</p>
+              </div>
+            </div>
+            <div class="check-result">
+              <div>
+                <p>Small Text</p>
+                <div class="pass">Icon</div>
+                <div class="pass">Icon</div>
+              </div>
+            </div>
+            <div class="check-result">
+              <div>
+                <p>Large Text</p>
+                <div class="fail">Icon</div>
+                <div class="fail">Icon</div>
+              </div>
+            </div>
+            <div class="check-result">
+              <div>
+                <p>Graphics</p>
+                <div class="pass">Icon</div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="secondary">
+          <h2>H2 Notes or "How-To-Use" section</h2>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt optio assumenda animi sint. Illo veritatis quos vitae quam provident accusamus. Atque dolor minus asperiores hic consequatur deserunt odit accusamus nisi repudiandae illum natus harum, temporibus mollitia eligendi pariatur molestiae dicta excepturi quo maxime! Mollitia excepturi libero autem molestiae quisquam adipisci.</p>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt optio assumenda animi sint. Illo veritatis quos vitae quam provident accusamus. Atque dolor minus asperiores hic consequatur deserunt odit accusamus nisi repudiandae illum natus harum, temporibus mollitia eligendi pariatur molestiae dicta excepturi quo maxime! Mollitia excepturi libero autem molestiae quisquam adipisci.</p>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt optio assumenda animi sint. Illo veritatis quos vitae quam provident accusamus. Atque dolor minus asperiores hic consequatur deserunt odit accusamus nisi repudiandae illum natus harum, temporibus mollitia eligendi pariatur molestiae dicta excepturi quo maxime! Mollitia excepturi libero autem molestiae quisquam adipisci.</p>
+        </section>
+      </div>
+    </main>
+
+    <footer>
+      <div class="container">
+        <p>&copy; 2022</p>
```

Comment:
The date should be added dynamically using JavaScript so it always stays current and we don't have to manually update it every time. 

Adding JavaScript to update the date will be broken up into a seperate ticket for someone else to work on. 

But for now, please make this change:
```suggestion
<!--TODO: Use JavaScript to dynamically add the current year here-->
        <p>&copy;</p>
```